middleware/src/middleware.go

```diff
@@ -1132,13 +1118,9 @@ func (middleware *Middleware) GetBaseInfo() rpcmessages.GetBaseInfoResponse {
 		ErrorResponse: &rpcmessages.ErrorResponse{
 			Success: true,
 		},
-		Status:                    "-PLACEHOLDER-", // FIXME: This is a placeholder.
-		Hostname:                  hostname,
 		MiddlewareLocalIP:         middlewareIP,
 		MiddlewarePort:            middlewarePort,
 		MiddlewareTorOnion:        middlewareTorOnion,
-		IsTorEnabled:              isTorEnabled,
```

Comment:
Also remove the `middleware.redisClient.GetBool(redis.TorEnabled)` query above?

Reply:
It's used for `middlwareTorOnion` in the lines after.